darshan-util/darshan-logutils.h

```diff
@@ -71,6 +71,7 @@ struct darshan_mod_info
     int  len;
     int  ver;
     int  idx;
+    int partial_flag;
```

Comment:
double checking: is this expected for merging back into `main`? (along with other changes in `darshan-util`)

Reply:
looks ok to me, assuming the python bindings match this change too (I assume they do).

Reply:
Yeah, this was the infrastructure needed to pass the module "partial_flag" up to PyDarshan so that we could notify about modules that were missing data. It was only implemented in pydarshan-devel, so this is expected in the merge.